test/cqlpy/test_sstable_validation.py

```diff
@@ -177,7 +190,8 @@ def test_scylla_sstable_validate_mismatching_partition(cql, scylla_path, temp_wo
             version = version,
             error_message = "mismatching index/data: partition mismatch")
 
-@pytest.mark.parametrize("version", versions)
+# FIXME: this should pass on ms too, no?
```

Comment:
I think we are not really prepared for `ms` sstables validation:
https://github.com/scylladb/scylladb/blob/7586c5ccbd3b9a8214e0f7d93a44fbf98ceda17e/sstables/sstables.cc#L2167-L2169

Here, the code will take the m[ce] specific path, which won't work for trie indexes.

Reply:
We can mix up the partition sizes, no? The goal here isn't to detect every possible index/sstable mixup, just common cases. Commonly the partition sizes won't match.